c2corg_api/models/route.py

```diff
@@ -169,6 +169,21 @@ class _RouteMixin(object):
     'mtb_height_diff_portages', 'rock_types', 'climbing_outdoor_type',
     'slackline_type', 'slackline_height']
 
+# List the ratings associated that can search by the outings
+rating_activites_associations = {
```

Comment:
Why do you need this?

Reply:
I am a bit surprised because I don't think we have to make changes in the routes, only in the outings.

Reply:
Removed after todays meeting =)